packages/react-vapor/src/components/table-hoc/TableWithDatePicker.tsx

```diff
@@ -79,7 +79,7 @@ export const tableWithDatePicker = (supplier: ConfigSupplier<ITableWithDatePicke
             const datePickerAction = (
                 <DatePickerDropdownConnected
                     {...(config as any)}
-                    id={this.props.id}
+                    id={`${this.props.id}_datePicker`}

```

Comment:
Pretty sure this is a breaking change. Is `TableHOCUtils.getDatePickerId` still working?

This util is defined as 
```ts
const getDatePickerId = (tableId: string) => `${tableId}-date-range`;
```

Reply:
![Screenshot from 2019-11-04 17-10-46](https://user-images.githubusercontent.com/45688129/68162576-587df280-ff26-11e9-86d8-212d2f08b726.png)

Reply:
since the feature seems well tested and that the test pass. Do you know how this could be a breaking change ? I see you are using `getDatePickerId` in the `urlTable`

Reply:
I'm pretty sure the tests in `TableWithDatePicker.spec.tsx` doesn't validate the id of the `DatePickerDropdownConnected`, this explains why it's not failing

Reply:
You don't have to modify this id if you already changed the on in SecondaryActions.tsx